File: src/BukkitPE/command/CommandSender.php
<?php
namespace BukkitPE\command;

use BukkitPE\permission\Permissible;

interface CommandSender extends Permissible{

	/**
	 * @param string $message
	 */
	public function sendMessage($message);

	/**
	 * @return \BukkitPE\Server
	 */
	public function getServer();

	/**
	 * @return string
	 */
	public function getName();


}

File: src/net/BukkitPE/command/CommandSender.java
package net.BukkitPE.command;

import net.BukkitPE.Server;
import net.BukkitPE.event.TextContainer;
import net.BukkitPE.permission.Permissible;

public interface CommandSender extends Permissible{
	/**
     * Sends a message to the command sender.
     */
    void sendMessage(String message);

    /**
     * Sends a message to the command sender.
     */
    void sendMessage(TextContainer message);

    /**
     * Returns the server of the command sender.
     */
    Server getServer();

    /**
     * Returns the name of the command sender.
     */
    String getName();

	
	boolean isPlayer();
}
